main/tests/cypress/cypress/integration/create-project/create_project.spec.js

```diff
@@ -1,78 +1,114 @@
 describe(__filename, function () {
-	it('Test the create project page', function () {
-		cy.visitOpenRefine();
-		cy.navigateTo('Create Project');
-		cy.get('#create-project-ui-source-selection-tabs > div').contains('This Computer').click();
-		cy.get('.create-project-ui-source-selection-tab-body.selected').contains('Locate one or more files on your computer to upload');
+    it('Test the create project page', function () {
+        cy.visitOpenRefine()
+        cy.navigateTo('Create Project')
+        cy.get('#create-project-ui-source-selection-tabs > div')
+            .contains('This Computer')
+            .click()
```

Comment:
I'm not sure splitting this into multiple lines makes it more readable (which should be the primary goal).

What do others think?

Reply:
This is done by the Prettier, automated javascript beautifier, splitting into multiple lines makes you clear of the multiple invocations done on the grabbed element.

Reply:
The version on multiple lines seems okay to me, I wouldn't actively fight it if it's done by a reasonably standard bit of tooling. For me it's mostly a discussion up to @kushthedude and @fgiroud who will be the ones writing most content in this style in the coming months, so if both of them agree on a style, it works for me.

Reply:
Agreed, whatever options, it works for me about formatting